internal/services/oracle/oracle_autonomous_database_clone_from_database_resource.go

```diff
@@ -0,0 +1,491 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package oracle
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-03-01/autonomousdatabases"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/oracle/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+var _ sdk.ResourceWithCustomizeDiff = AutonomousDatabaseCloneFromDatabaseResource{}
+
+type AutonomousDatabaseCloneFromDatabaseResource struct{}
+
+type AutonomousDatabaseCloneResourceModel struct {
```

Comment:
Maybe use name that follows the pattern of `AutonomousDatabaseCloneFromBackupResourceModel`? 
```suggestion
type AutonomousDatabaseCloneFromDatabaseResourceModel struct {
```